color/quoted.c

```diff
@@ -122,3 +122,402 @@ bool quoted_colors_parse_color(enum ColorId color, uint32_t fg, uint32_t bg,
   *rc = MUTT_CMD_SUCCESS;
   return true;
 }
+
+/**
+ * qclass_free - Free a single QClass
+ * @param ptr QClass to free
+ */
```

Comment:
We should note in the docstring that `qclass_free_tree()` should be called instead of `qclass_free()` otherwise we'll leak memory. I can easily see contributors making this mistake and it slipping past review.

Reply:
see above